clms-api/src/main/java/com/clms/api/common/web/projections/GenericProjectionConverter.java

```diff
@@ -0,0 +1,5 @@
+package com.clms.api.common.web.projections;
+
+public interface GenericProjectionConverter<F, T> {
+    T convert(F from);
+}
```

Comment:
Generic projection converter interface, 

Services/Components can implement this so we have a defined API for converting entities to projections